templates/feed/feed-item.php

```diff
@@ -209,7 +209,7 @@
 	<pubDate><?php echo $pub_date; ?></pubDate>
 	<dc:creator><![CDATA[<?php echo $author; ?>]]></dc:creator>
 	<guid isPermaLink="false"><?php the_guid(); ?></guid>
-	<description><![CDATA[<?php echo $description; ?>]]></description>
+	<description><![CDATA[ <?php echo '<img src="' . esc_url( $episode_image ) . '" />'; ?> <?php echo $description; ?>]]></description>
```

Comment:
By removing the tags <image></image> the Feed loses the artwork of each episode.
With this line, it has the artwork back working, without violating the W3 requirements